packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx

```diff
@@ -1,115 +1,95 @@
-import {defineType} from '@sanity/types'
-import {fireEvent, render, screen, within} from '@testing-library/react'
+import {act, fireEvent, render, screen, within} from '@testing-library/react'
 import {route, RouterProvider} from 'sanity/router'
 import {beforeEach, describe, expect, it, vi} from 'vitest'
 
-import {getAllByDataUi, getByDataUi} from '../../../../../../test/setup/customQueries'
+import {getByDataUi} from '../../../../../../test/setup/customQueries'
 import {createTestProvider} from '../../../../../../test/testUtils/TestProvider'
+import {DefaultPreview} from '../../../../components/previews/general/DefaultPreview'
+import {
+  activeASAPRelease,
+  archivedScheduledRelease,
+  scheduledRelease,
+} from '../../../__fixtures__/release.fixture'
 import {releasesUsEnglishLocaleBundle} from '../../../i18n'
-import {type ReleaseDocument} from '../../../index'
-import {type DocumentHistory} from '../documentTable/useReleaseHistory'
 import {ReleaseSummary, type ReleaseSummaryProps} from '../ReleaseSummary'
 import {type DocumentInRelease} from '../useBundleDocuments'
-
-vi.mock('../../../../studio/addonDataset/useAddonDataset', () => ({
-  useAddonDataset: vi.fn().mockReturnValue({client: {}}),
+import {
+  documentsInRelease,
+  useBundleDocumentsMockReturnWithResults,
+} from './__mocks__/useBundleDocuments.mock'
+
+vi.mock('../../../index', () => ({
+  useDocumentPresence: vi.fn().mockReturnValue({
+    user: '',
+    path: '',
+    sessionId: '',
+    lastActiveAt: '',
+  }),
+  useDocumentPreviewStore: vi.fn().mockReturnValue({
+    unstable_observeDocumentIdSet: vi.fn(() => ({
+      pipe: vi.fn(),
+    })),
+  }),
 }))
 
-vi.mock('../../../../store', async (importOriginal) => ({
-  ...(await importOriginal()),
-  useUser: vi.fn().mockReturnValue([{}]),
+vi.mock('../useBundleDocuments', () => ({
+  useBundleDocuments: vi.fn(() => useBundleDocumentsMockReturnWithResults),
 }))
 
-vi.mock('../../../../user-color', () => ({
-  useUserColor: vi.fn().mockReturnValue('red'),
-}))
+vi.mock('../../../../studio/components/navbar/search/components/SearchPopover')
 
-const timeNow = new Date()
+vi.mock('../../../../preview/components/_previewComponents', async () => {
+  return {
+    _previewComponents: {
+      default: vi.fn((arg) => <DefaultPreview {...arg} />),
```

Comment:
Could this be a dummy component or are we asserting anything "default preview" specific?

Reply:
We are asserting on the passing through and rendering of the document preview itself. this mock only exists as the `_previewComponents` has an as yet unknown quirk that during test runs all the exports are set to `undefined` rather than their correct components